src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
-public class Application {
+public class Application extends Lottery {
```

Comment:
we shouldn't extend Lottery class here
```suggestion
public class Application {
```